### PR TITLE
add tokenContractAddress to UserAction

### PR DESCRIPTION
**Pull request by daiki44**
## What?
- add tokenContractAddress to UserAction
- ActionType to string enum
- export ActionType, UserAction
- use JS native types as public interface (UserAction)
WAK-320

### CODE REVIEW

File: packages/plasma-light-client/src/managers/TokenManager.ts [hunk 1]
```diff
@@ -104,6 +104,20 @@ export default class TokenManager {
     return this.tokenContracts.get(tokenContractAddress.data)
   }
 
+  /**
+   * Get Token Contract by deposit contract address.
+   * @param depositContractAddress The address of deposit Contract
+   */
+  getTokenContractByDepositContractAddress(
```
Comment (daiki44) [inline packages/plasma-light-client/src/managers/TokenManager.ts:111]: @tkmct I want to get the token contract address by deposit contract address.
What do you think is the best way?

File: packages/plasma-light-client/src/managers/TokenManager.ts [hunk 2]
```diff
@@ -15,7 +15,8 @@ export default class TokenManager {
   private tokenNames: Map<string, string> = new Map() // key: tokenContractAddress, value: name
   private tokenSymbols: Map<string, string> = new Map() // key: tokenContractAddress, value: symbol
   private tokenDecimals: Map<string, Integer> = new Map() // key: tokenContractAddress, value: decimals
-  private contractAddressMap: Map<string, string> = new Map() // key: tokenContractAddress, value: depositContractAddress
+  private tokenContractAddressMap: Map<string, string> = new Map() // key: tokenContractAddress, value: depositContractAddress
+  private depositContractAddressMap: Map<string, string> = new Map() // key: depositContractAddress, value: tokenContractAddress
```
Comment (tkmct) [inline packages/plasma-light-client/src/managers/TokenManager.ts:19]: can you name more verbose?
~to~ is good name for mapping.

Comment (daiki44) [reply packages/plasma-light-client/src/managers/TokenManager.ts:19]: Like this?
```
tokenContractAddressToDepositContractAddressMap
depositContractAddressToTokenContractAddressMap
```

Comment (tkmct) [reply packages/plasma-light-client/src/managers/TokenManager.ts:19]: `tokenToDeposit`, `depositToToken`?

**Pull request merged**